src/core/network/client.ts

```diff
@@ -320,7 +321,7 @@ class ApiClient {
   /**
    * If set to `true`, first _eligible_ call will avoid performing a POST request.
    * Note that such a call:
-   * 1. cannot report errors,
+   * 1. cannot report errors or return meaningful value,
```

Comment:
Sounds like we're talking about human values, when we actually mean a piece of memory

```suggestion
   * 1. cannot report errors or return a meaningful value,
```

</nitpick>